src/pipeline_edsl/to_workflow.ml

```diff
@@ -21,6 +21,7 @@ module File_type_specification = struct
     | Fastqc_result: list_of_files workflow_node -> [ `Fastqc ] t
     | Isovar_result: single_file workflow_node -> [ `Isovar ] t
     | Topiary_result: single_file workflow_node -> [ `Topiary ] t
+    | Vaxrank_result: Vaxrank.product workflow_node -> [ `Vaxrank ] t
```

Comment:
Here we need a full module path:
```ocaml
    | Vaxrank_result: Biokepi_bfx_tools.Vaxrank.product workflow_node -> [ `Vaxrank ] t
```

Problem of the `OMakeroot` I guess that let's that problem build anyway and it fails when we use the type for extending the EDSL.